### DISCUSSION TITLE
No more webradios in folder trees with depth > 1

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#1786] [Fixed Issues]
I organized my webradios in a folder structure. Webradios inside a nested folder e.g. webradio/level1/level2 are no more present in the GUI, but present in the file system. Creation of a new webradio in a level2 folder is no more possible. 
My workaroud: Until a solution is available I copied (using ssh/bash) all level2 webradio files to the parent (level1) directory.

**Comment by rern** [comment]
Please try the UPDATE.

**Reply by klassikmann** [reply]
@rern 
Solved. Thank you.
